lib/util/versioning.js

```diff
@@ -55,6 +69,8 @@ function get_runtime_abi(runtime, target_version) {
     }
     if (runtime === 'node-webkit') {
         return get_node_webkit_abi(runtime, target_version || process.versions['node-webkit']);
+    } else if (runtime === 'electron') {
+        return get_electron_abi(runtime, target_version || process.versions.electron);
```

Comment:
Oh this is a different error than I thought it was - I'll look into this. Can you file a bug on electronjs/electron-rebuild?